wp-modules/editor/js/src/utils/toKebabCase.ts

```diff
@@ -1,7 +1,5 @@
 /** Converts a string to a kebab-case. */
 export default function toKebabCase( toConvert = '' ) {
-	return toConvert
-		.replace( /[_\W]+(?=\w+)/g, '-' )
```

Comment:
Because this stripped all non-word characters, it caused a category with an apostrophe [to not appear](https://github.com/studiopress/pattern-manager/blob/try/add-new-categories-in-editor-ui/wp-modules/editor/js/src/utils/addNewCategory.ts#L22) the `<Select>`.

With the [PHP edits](https://github.com/studiopress/pattern-manager/pull/200/files#diff-e9d8f99a8c7ab5fa15a0fb8c0c3b3c942b1d7d7fdb1dd49794453762dbadd8fdR398), categories could have an apostrophe.

Reply:
```php
<?php
/**
 * Title: My New Pattern 2
 * Slug: altitude-pro/my-new-pattern-2
 * Description: 
 * Categories: you-re-the-best        <-- Saved without the apostrophe, it was stripped in CategoriesPanel.ts#L62
 * Keywords: 
 * Viewport Width: 1280
 * Block Types: 
 * Post Types: 
 * Inserter: true
 * Custom Categories: You're The Best
 */
// Category name and label have the apostrophe
register_block_pattern_category( 'you\'re-the-best', [ 'label' => __( 'You\'re The Best', 'altitude-pro' ), 'pm_custom' => true ] ); 
?>


```